app/js/components/delete_member.ts

```diff
@@ -11,8 +11,8 @@ $(() => {
       if (!target.matches('.teams__deleteMemberLink')) return;
       event.preventDefault();
 
-      const td = document.querySelector('.teams__actions') as HTMLTableDataCellElement;
-      td.classList.add('loading');
+      const td = $(target).closest('td');
+      td.addClass('loading');
```

Comment:
Why are you reverting to jQuery here?  We were phasing out it's usage when I was working on it, has that decision been reversed?

Reply:
You can likely use the target. variable, defined on line 10 to navigate to the closest td and delete it without using jQuery?

Reply:
I was not informed that jquery was being phased out, I will revert this back to regular js.